### ISSUE TITLE
Add proper support for non-QWERTY layouts in keybindings

### ISSUE THREAD

**Issue by andrhua** [mbnuqw/sidebery#476]
## Description
![image](https://user-images.githubusercontent.com/15843809/134586876-4da533c2-f5a2-4994-ab4b-75d3701b5a45.png)
When setting `Ctrl + S` on Colemak, extension registers it as `Ctrl + D`, and then when using this shortcut I should press `Ctrl + D` on Colemak, which is `Ctrl + G` on QWERTY.

I think using [`key`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) instead of `code` [here](https://github.com/mbnuqw/sidebery/blob/bbd90548afe76db40c3923cf083c51d2a40be3ab/src/page.settings/components/settings.keybindings.vue#L107) should do the trick.

**Comment by freMea** [comment]
Yes I use AZERTY and the problem is it detects the right keybinding but always display the as it is in a QWERTY.

**Comment by mbnuqw** [comment]
Fixed from v5.0.0
